en, zh: update doc for backup and restore (#233)
pingcap/docs-tidb-operator: en/backup-to-s3.md

### Ad-hoc backup process

> **Note:**
>

> > * If need to add `Prefix` to the backup data path, you can add the `Prefix` in the `bucket` configuration, e.g. `bucket: demo-bucket/demo-prefix`

Reviewer: ```suggestion
> * If you need to add `Prefix` to the backup data path, configure the `Prefix` in the `bucket` configuration in the YAML file, e.g.: `bucket: demo-bucket/demo-prefix`
```

> > * Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you have to configure `spec.s3.options` as below in the yamls used in the following procedures, for more detail, refer to the [document](https://rclone.org/s3/#key-management-system-kms):

Reviewer: ```suggestion
> * Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you need to configure `spec.s3.options` in the YAML file in the following steps:
```

>
>     ```yaml
>     spec:
>       ...
>       s3:
>         ...
>         options:
>         - --ignore-checksum

> 

Reviewer: ```suggestion
>     ```
>
> * For more details, refer to the [rclone | Key Management System](https://rclone.org/s3/#key-management-system-kms).

```

        secretName: backup-demo1-tidb-secret
    s3:
        provider: aws
        # region: us-east-1
        region: ${region}
        bucket: ${bucket}
        # storageClass: STANDARD_IA
        # acl: private
        # endpoint:
    storageClassName: local-storage
    storageSize: 10Gi
    ```

> In the above two examples, all data of the TiDB cluster is exported and backed up to Amazon S3 and Ceph respectively. You can ignore the `region`, `acl`, `endpoint`, and `storageClass` configuration items in the Amazon S3 configuration. S3-compatible storage types other than Amazon S3 can also use configuration similar to that of Amazon S3. You can also leave the configuration item fields empty if you do not need to configure these items as shown in the above Ceph configuration.

Reviewer 2: I think ceph only need to fill endpoint

Author: Ceph also supports acl, storageClass, etc.

### Scheduled backup process

> **Note:**
>

> > * If need to add `Prefix` to the backup data path, you can add the `Prefix` in the `bucket` configuration, e.g. `bucket: demo-bucket/demo-prefix`

Reviewer: ```suggestion
> * If you need to add `Prefix` to the backup data path, configure the `Prefix` in the `bucket` configuration in the YAML file, e.g.: `bucket: demo-bucket/demo-prefix`
```

> > * Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you have to configure `spec.backupTemplate.s3.options` as below in the yamls used in the following procedures, for more detail, refer to the [document](https://rclone.org/s3/#key-management-system-kms):

Reviewer: ```suggestion
> * Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you need to configure `spec.backupTemplate.s3.options` in the YAML file in the following steps:
```

>
>     ```yaml
>     spec:
>       ...
>       backupTemplate:
>         ...
>         s3:
>           ...
>           options:
>           - --ignore-checksum

> 

Reviewer: ```suggestion
>     ```
>
> * For more details, refer to the [rclone | Key Management System](https://rclone.org/s3/#key-management-system-kms).

```
